src/modules/UI/scenes/Settings/action.js

```diff
@@ -103,6 +105,7 @@ export const setDefaultFiatRequest = (defaultFiat: string) => (dispatch: Dispatc
       ACCOUNT_SETTINGS.setSpendingLimits(account, nextSpendingLimits)
       // update spending limits in settings
       dispatch(newSpendingLimits(nextSpendingLimits))
+      dispatch(updateExchangeRates())
     })
```

Comment:
Since the default fiat has changed that means that the necessary exchange rates must be re-built